Damaris-Landers-ManchesterUK.md

```diff
@@ -0,0 +1,16 @@
+# Manchester, UK
+
+### Restaurants
+
+- Black Dog Ballroom
```

Comment:
@Damaris-Landers  I'll check it next time I pass by Manchester. Does Ballroom means there is dancing as well?